reactioncommerce/reaction
# GraphQL remove account from group mutation

Resolves #3926   
Impact: **minor**  
Type: **feature**

## Issue

Adds the `removeAccountToGroup` GraphQL mutation

## Solution

Add a mutation to remove an account from a group. The mutation is very similar to other mutations so it was mostly copy-paste.

Update the `group/removeUser` method to return the group the user was added to as the result of them being removed from their current group instead of `{ status: 200 }`.

## Breaking changes

**Possible**

Update the `group/removeUser` method to return the group the user was added to as the result of them being removed from their current group instead of `{ status: 200 }`. It doesn't seem that any of the places where this method was called cared about the result, only that it exists.

## Testing
1. As an admin, invite a user
1. In the database, find the user you created and get the ID
1. Encode the account id like so `reaction/account:<accountId>` as a utf8 base64 string
1. Also, take note of the ID of the group the user is a member of. It should be the "shop manager" group
1. Encode the group id like so `reaction/group:<groupId>` as a utf8 base64 string
1. Run the following mutation with the id's you generated

```
mutation {
    removeAccountFromGroup(input: {
    accountId: "b64AccountId"
    groupId: "b64GroupId"
  }) {
    clientMutationId,
    group {
      name
      _id
    }
}
}
```

8. See the result says has `name: customer`, as the user should now be a part of the customer group.
